src/imagekit.cs

```diff
@@ -199,19 +197,17 @@ interface IKDeviceBrowserView {
 		[Export ("mode")]
 		IKDeviceBrowserViewDisplayMode Mode { get; set;  }
 
-//		FIXME need ImageCaptureCore;
-//		[Export ("selectedDevice")]
-//		ICDevice SelectedDevice { get;  }
+		[Export ("selectedDevice")]
+		ICDevice SelectedDevice { get;  }
 	}
 
 	[BaseType (typeof (NSObject))]
 	[Model]
 	[Protocol]
 	interface IKDeviceBrowserViewDelegate {
-//		FIXME need ImageCaptureCore;
-//		[Abstract]
-//		[Export ("deviceBrowserView:selectionDidChange:"), EventArgs ("IKDeviceBrowserViewICDevice")]
-//		void SelectionDidChange (IKDeviceBrowserView deviceBrowserView, ICDevice device);
+		[Abstract]
```

Comment:
```suggestion
#if XAMCORE_4_0
		[Abstract]
#endif
```

New members in existing protocols can't be `[Abstract]`, it's a breaking change.

Reply:
IIRC that was discussed in the original PR

Reply:
Yes, here it is

https://github.com/xamarin/xamarin-macios/pull/4404#issuecomment-407481964

> I have undone the XAMCORE_4_0 change, as this API while breaking change could have never been used by anyone before, so the regular policy does not apply to this.